tests/msg_queue_print/tests/01-run.py

```diff
@@ -8,17 +8,21 @@
 
 # @author      Julian Holzwarth <julian.holzwarth@fu-berlin.de>
 
+import os
 import sys
 from testrunner import run
 
 
 def testfunc(child):
     child.expect("No message queue")
-    child.expect("Message queue of thread 2")
+    child.expect(r"Message queue of thread \d+")
     child.expect_exact('size: 8 (avail: 0)')
-    child.expect("Message queue of thread 2")
+    child.expect(r"Message queue of thread \d+")
     child.expect_exact('size: 8 (avail: 8)')
-    child.expect_exact('type: 0x0000, content: 0 ((nil))')
+    if os.environ.get('BOARD') == 'native':
+        child.expect_exact('type: 0x0000, content: 0 ((nil))')
+    else:
+        child.expect_exact('type: 0x0000, content: 0 (0)')
```

Comment:
IIRC there are also variants where a NULL pointer can be any variant of `0x0+` depending on the toolchain. Maybe reflect that with a regex?
```suggestion
        child.expect(r'type: 0x0000, content: 0 \((0x)?0+\)')
```

Reply:
Started a run https://ci.inria.fr/ci-riot-tribe/job/build-pipeline.jk/268/, lets see if anything is not covered.

Reply:
All BOARDs passed except two that failed during flash, so unrelated.